nightingale_perception/config/movo_body_filter.yaml

```diff
@@ -0,0 +1,25 @@
+cloud_filter_chain:
+- name: body_filter
+  type: robot_body_filter/RobotBodyFilterPointCloud2
+  params:
+     #frames/fixed: "odom"
+     frames/sensor: "camera_link" # might be camera_depth_optical_link
```

Comment:
does the data published by the camera node list the frame as camera_link or is it camera_depth_optical_link? i think this should match whichever it is?

Reply:
Got catch the frame in the pointcloud data is "camera_depth_optical_frame". I'll be more thorough by making a checklist and keeping better tabs on things I need to do before PR sorry.